### PR TITLE
[acsone-addons][add] Mail Html Wigedt Template to manage placeholder

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
# Mail HTML Widget Template

This module allow to immediately manage placeholder from the mail composer
This tools is available on each html widget of Odoo
- Primary dropdownlist is filled with expected fields of the object associated
  to the template.
- When choosing a field in a dropdownlist, two possibilities:
  - If the selected field is a relation to another object, an additional
    dropdownlist filled with the expected fields of this object is append
    to the toolbar (this generic mechanism follows relations between objects
    through the data model and is intended for an undefined number of
    levels)
  - Otherwise the placeholder is inserted into textarea

### CODE REVIEW

**Comment by lmignon** [comment]
:+1:

File: mail_html_widget_template/__init__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import wizard
```
Comment (laetitia-gangloff) [inline mail_html_widget_template/__init__.py]: I think it is better to use from . import wizard

File: mail_html_widget_template/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        'email_template',
+    ],
+    'description': """
+Mail HTML Widget Template
+=========================
+This module allow to immediately manage placeholder from the mail composer
+This tools is available on each html widget of Odoo
+
+* Primary dropdownlist is filled with expected fields of the object associated
+    to the template.
```
Comment (laetitia-gangloff) [inline mail_html_widget_template/__openerp__.py]: strange alignement, no ?

File: mail_html_widget_template/static/src/xml/mail_html_widget_template.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        <div t-att-style="'border-bottom:thin solid'" class="custom_cleditorToolbar" id="selector_toolbar">
+        </div>
+    </t>
+    <t t-name="ModelFieldsSelection" size="5">
+        <select class="oe_model_field_selector" id="new_select">
+            <option></option>
+        </select>
+    </t>
+
+</templates>
```
Comment (laetitia-gangloff) [inline mail_html_widget_template/static/src/xml/mail_html_widget_template.xml]: new line at end ?

**Comment by laetitia-gangloff** [comment]
:+1:

**Comment by anthony-muschang** [comment]
:+1:

**Pull request merged**